src/ompl/tools/thunder/Thunder.h

```diff
@@ -147,6 +148,15 @@ namespace ompl
                 return solve(ptc);
             }
 
+            base::PlannerStatus solve(const base::PlannerTerminationCondition &ptc, const std::size_t minSolCount,
+                                      const std::size_t maxSolCount, const bool hybridize)
+            {
+                hybridize_ = hybridize;
+                minSolCount_ = minSolCount;
+                maxSolCount_ = maxSolCount;
+                return solve(ptc);
+            }
+
```

Comment:
why is this overload necessary? docstring is missing

Reply:
i added a docstring but i'm not sure it answers your question